pinot-core/src/test/java/com/linkedin/pinot/index/reader/SortedIndexReaderTest.java

```diff
@@ -16,36 +16,47 @@
 package com.linkedin.pinot.index.reader;
 
 import com.linkedin.pinot.common.segment.ReadMode;
+import com.linkedin.pinot.common.utils.Pairs;
 import com.linkedin.pinot.core.io.reader.impl.v1.SortedIndexReader;
 import com.linkedin.pinot.core.io.writer.impl.FixedByteSingleValueMultiColWriter;
 import com.linkedin.pinot.core.segment.memory.PinotDataBuffer;
 import java.io.File;
+import java.io.IOException;
 import java.nio.channels.FileChannel;
 import java.util.Random;
+import org.apache.commons.io.FileUtils;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
-@Test
-public class SortedForwardIndexReaderTest {
-  private static Logger LOGGER = LoggerFactory.getLogger(SortedForwardIndexReaderTest.class);
-  public void testSimple() throws Exception {
 
+public class SortedIndexReaderTest {
+  private static Logger LOGGER = LoggerFactory.getLogger(SortedIndexReaderTest.class);
+
+  private int cardinality = 100000;
```

Comment:
Since you refactored the test, please follow the code style convention for variables ("_cardinality")